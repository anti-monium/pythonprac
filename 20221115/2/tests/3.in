class C:
  num = Num()
  
print(C().num)
c, d = C(), C()
c.num = d.num = 17
print(c.num+d.num)
c.num = "jrlgglwaipu;env   ppe"
print(c.num+d.num)
d.num = range(10, 1312, 7)
print(c.num+d.num)
c.num = [1,2,3,4,5,6,7,8,9,0,3,4,5,7,7,8,9,6,5,3,2,1,3,6,7,8]
d.num = (1,)
print(c.num+d.num)
