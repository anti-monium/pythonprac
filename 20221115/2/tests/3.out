0
34
38
207
27
